frontend/src/metabase/ui/components/inputs/TextInputBlurChange/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { TextInput } from "@mantine/core";
```

Comment:
`InputBlurChange` has a unit test suite - let's copy it for `TextInputBlurChange`, it should work the same.

Reply:
Done.